src/systemTest/java/com/flightstats/hub/system/functional/WebhookErrorTest.java

```diff
@@ -144,6 +144,13 @@ void testSettingCursorBeyondErrorClearsErrorStateAndContinues() {
         // verify that no errors exist on the hub
         log.info("Verifying that no errors exist on the hub for webhook {}", webhookName);
         assertTrue(callbackService.isErrorListEventuallyCleared(webhookName));
+
+        log.info("deleting " + channelName + " and " + webhookName);
```

Comment:
This should be parameterized :)
log.info("deleting {} and {}", channelName , webhookName);